api_spec.yaml

```diff
@@ -1266,17 +1472,285 @@ paths:
+                        schema:
+                            type: object
+                            properties:
+                                tenantId:
+                                    type: string
+                                userId:
+                                    type: string
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                oneOf:
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "OK"
+                                          wasAlreadyAssociated:
+                                              type: boolean
+                                              default: false
+
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "UNKNOWN_USER_ID_ERROR"
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "EMAIL_ALREADY_EXISTS_ERROR"
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "PHONE_NUMBER_ALREADY_EXISTS_ERROR"
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "THIRD_PARTY_USER_ALREADY_EXISTS_ERROR"
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "ASSOCIATION_NOT_ALLOWED_ERROR"
+                                          reason:
+                                              type: string
+
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+    /dashboard/api/tenants/user/disassociate:
```

Comment:
remove this api, no need to implement it now

Reply:
Done